hopp/simulation/technologies/steel/enthalpy_functions.py

```diff
@@ -0,0 +1,194 @@
+## Functions to calculate enthalpy of different streams 
+
+### Hydrogen enthalpy coefficients
+# The value of enthalpy returned is the specific enthalpy in kj/g
+'''
+Author: Charlie Kiefer
+Date: 8/14/23
+
+These functions calculate enthalpies of elements relating to green steel
+
+Sources:
+Chase, M.W., Jr. 1998. "NIST-JANAF Themochemical Tables, Fourth Edition." J. Phys. Chem. Ref. Data, Monograph 9 1-1951. doi:https://doi.org/10.18434/T4D303.
+    Thermochemistry tab of each element
+
+Inputs:
+    T: Temperature of the element (K)
+
+Outputs:
+    H_t: Enthalpy of element at given temp (kj/g)
+'''
+
+def h2_enthalpy_1(T):
```

Comment:
I don't see any tests for these functions.

Reply:
Added tests for these functions.  These functions could be added to but will take time to implement all